python/lammps.py

```diff
@@ -703,6 +708,21 @@ def get_natoms(self):
     return int(self.lib.lammps_get_natoms(self.lmp))
 
   # -------------------------------------------------------------------------
+  
+  # -------------------------------------------------------------------------
+
+  def get_nbonds(self):
+    """Get the total number of bonds in the LAMMPS instance.
+
+    Will be precise up to 53-bit signed integer due to the
+    underlying :cpp:func:`lammps_get_nbonds` function returning a double.
+
+    :return: number of bonds
+    :rtype: int
+    """
+    return int(self.lib.lammps_get_nbonds(self.lmp))
+    
+  # -------------------------------------------------------------------------
```

Comment:
This description doesn't match up with the library implementation, which returns an `int`.

Reply:
I have removed `get_nbonds()` and used `get_thermo('bonds')` in the example to get the number of bonds which makes this function obsolete.